src/components/sortable/dragLayer/dragLayer.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { describe, it, expect } from 'vitest';
+import { renderWithDnd } from '@common/utils/testUtils';
+import { DragLayer } from './dragLayer';
+import type { DragItem } from '@common/types';
```

Comment:
Please organize imports this way in the files in this PR
```suggestion
import { describe, it, expect } from 'vitest';

import { renderWithDnd } from '@common/utils/testUtils';
import type { DragItem } from '@common/types';

import { DragLayer } from './dragLayer';
```

Reply:
Done!. 

Maybe we need the task tech, where include auto sorting imports?

Reply:
Agree, let's discuss it on Wednesday's sync